src/miscellaneous/data-visualization.tex

```diff
@@ -32,7 +32,7 @@ \subsection{Requirements}\label{subsec:requirements}
 sales increase during holiday periods.
 
 And in tandem with that lies the client's underlying objective to optimize their business.
-The data visualization should serve the purpose of letting the client understand how to optimize their café's food
+The data visualization should serve the purpose of letting the client understand how to optimize their caf\'e's food
```

Comment:
@Freja-bot Don't you get this warning in IntelliJ? You should use the unicode `é` instead of ASCII `\'e`:

<img width="501" alt="SCR-20241007-bdxu" src="https://github.com/user-attachments/assets/71e2fe33-4c59-4341-aec9-66d153018854">